logstash-core/build.gradle

```diff
@@ -15,12 +16,23 @@ repositories {
 buildscript {
     repositories {
         mavenCentral()
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
     }
     dependencies {
         classpath 'org.yaml:snakeyaml:1.17'
+        classpath "gradle.plugin.com.github.jk1:gradle-license-report:0.7"
     }
 }
 
+apply plugin: 'com.github.jk1.dependency-license-report'
+
+licenseReport {
```

Comment:
Shouldn't this depend on downloading a bunch of gems and setting up the env?

I think we probably need a task in the parent project that does those things then calls this.

Reply:
Doesn't this need an explicit `task` declaration? 

I'm not sure how to actually run this. Can you lend me some gradle expertise?

Reply:
Yes, you can run this via `./gradlew logstash-core:generateLicenseReport`. The output will be written to the `logstash-core/build/reports/dependency-license` directory.